doc/manual/src/command-ref/files/user-profiles.md

```diff
@@ -0,0 +1,157 @@
+# User profiles
+
+A directory that contains links to user profiles managed by [`nix-env`] and [`nix profile`].
+
+- `$NIX_STATE_DIR/profiles/per-user/$USER`
+- `${XDG_STATE_DIR-$HOME/.local/state}/nix/profiles` if [`use-xdg-base-directories`] is set to `true`
```

Comment:
I'll let you and @fricklerhandwerk be the judge over whether `${XDG_STATE_DIR-$HOME/.local/state}` or plain `$XDG_STATE_DIR` (and maybe a separate XDG reference page) is better, but regardless I think it would be good to be consistent.